lib/rs/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "rift"
```

Comment:
@anatol I've previously changed this - perhaps you accidentally looked at an older version of the PR?